docs/docs/configurations.md

```diff
@@ -0,0 +1,95 @@
+# Configuration Options
+
+NetKet exposes a few configuration options which can be set through environment variables by doing something like
+```bash
+# without exporting it
+NETKET_DEBUG=1 python
+
+# by exporting it
+export NETKET_DEBUG=1
+python ...
+
+# by setting it within python
+python
+>>> import os
+>>> os.environ["NETKET_DEBUG"] = "1"
+>>> import netket as nk
+```
+Some configuration options can also be changed at runtime by using the function {func}`netket.config.update(option_name, value)`, which can be used as:
+```python
+>>> import netket as nk
+>>> nk.config.update("NETKET_DEBUG", True)
+>>> ...
+```
+
+You can always query the value of an option by accessing the `nk.config` module:
+```python
+>>> import netket as nk
+>>> nk.config.netket_debug
+False
+>>> nk.config.update("NETKET_DEBUG", True)
+>>> nk.config.netket_debug
```

Comment:
is it possible to directly change this entry directly i.e. `nk.config.netket_debug=False` or one must use the update function?

Reply:
no, which is why I explain to use `config.update`.
We could make it directly updatable in the future. It's simple but not trivial, because some parts of the code that use those flags might need to be adapted slightly.

Reply:
yes it would be nice to make it directly accessible but ok, it's fine...

Reply:
ok, it was easier than expected. It's now possible to change them directly.